test/util/test_runners/integration_test_case.py

```diff
@@ -0,0 +1,71 @@
+"""
+extend the unitest TestCase class
+"""
+
+# Builtin
+import io
+import sys
+
+# 3rp parties
+import itertools
+from unittest import TestCase
+from contextlib import redirect_stdout
+import subprocess as sp
+from unittest.mock import patch, MagicMock
+
+# internals
+
+from conjur.cli import Cli
+from test.util.test_runners.params import ClientParams, TestEnvironmentParams
+
+
+class IntegratioTestCaseBase(TestCase):
+
+    def __init__(self, testname, client_params: ClientParams = None,
+                 environmentParams: TestEnvironmentParams = None):
+        """
+        Base class that extends unittest TestCase
+        used to setup tests environment
+        """
+        self.environment = TestEnvironmentParams() if environmentParams is None else environmentParams
+        self.client_params = ClientParams() if client_params is None else client_params
+
+        super(IntegratioTestCaseBase, self).__init__(testname)
+
+
+    def invoke_cli(self, *args, exit_code=0) -> str:
+        """
+        invoker for the integration tests
```

Comment:
let's make this comment aligned so line lengths are relatively even. This comment applies to all places where you have docs